IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -193,10 +193,19 @@ - (MSIDPrimaryRefreshToken *)getPrimaryRefreshTokenWithAccount:(MSIDAccountIdent
 - (NSArray<MSIDPrimaryRefreshToken *> *)getPrimaryRefreshTokensForConfiguration:(MSIDConfiguration *)configuration
                                                                         context:(id<MSIDRequestContext>)context
                                                                           error:(NSError *__autoreleasing*)error
+{
+    return [self getPrimaryRefreshTokensForConfiguration:configuration account:nil context:context error:error];
+}
+
+- (NSArray<MSIDPrimaryRefreshToken *> *)getPrimaryRefreshTokensForConfiguration:(MSIDConfiguration *)configuration
+                                                                        account:(MSIDAccountIdentifier *)accountIdentifier
+                                                                        context:(id<MSIDRequestContext>)context
+                                                                          error:(NSError *__autoreleasing*)error
 {
     MSIDDefaultCredentialCacheQuery *query = [MSIDDefaultCredentialCacheQuery new];
     query.environmentAliases = [configuration.authority defaultCacheEnvironmentAliases];
     query.credentialType = MSIDPrimaryRefreshTokenType;
+    query.homeAccountId = accountIdentifier.homeAccountId;
```

Comment:
since change a keychain query criteria, I think it'd be good to cover by UT to confirm its correctness.

Reply:
This function is about primary refresh token, it is tested in the broker PR file [ADBrokerTokenCacheAccessorIntegrationTests.m](https://github.com/AzureAD/azure-activedirectory-tokenbroker-for-objc/pull/1703/files#diff-539120df488024b2e9818e730d852cfc41d6c5c5b35148ec1782f47cae4d1e9f)